webapp/src/filtering/filtering.tsx

```diff
@@ -71,12 +104,22 @@ export function PageButtons() {
 
 type ReplayData = {
```

Comment:
I think when we get gql, we can import types, so you don't need to do this type definition here.